src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -656,13 +657,21 @@ public Collection<Research> getCivilResearch(Collection<Research> r)
           .addToList(r);
 
         // Primary Research #5
-        new Research(new ResourceLocation(Constants.MOD_ID, "civilian/ambition"), CIVIL).setTranslatedName("Ambition")
+        final Research ambition = new Research(new ResourceLocation(Constants.MOD_ID, "civilian/ambition"), CIVIL).setTranslatedName("Ambition")
           .setSortOrder(5)
           .setIcon(ModBlocks.blockHutMysticalSite.asItem())
           .addItemCost(Items.DIAMOND, 1)
           .addEffect(ModBuildings.mysticalSite.get().getBuildingBlock(), 1)
           .addToList(r);
 
+        new Research(new ResourceLocation(Constants.MOD_ID, "civilian/air"), CIVIL).setTranslatedName("Scuba")
+          .setParentResearch(ambition)
+          .setSortOrder(1)
+          .setIcon(Items.POTION)
+          .addItemCost(Items.HEART_OF_THE_SEA, 1)
```

Comment:
Heart of the Sea is a super-rare/expensive item for being a level 2 research...

Reply:
Huh? It's super easy to obtain via the treasure maps. And since it's an optional research I find it an okay cost